### PR TITLE
CNV-36963: Issue in file virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/CNV-36963
OCP 4.14+
Preview: https://72111--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge

### CODE REVIEW

File: virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc [hunk 1]
```diff
@@ -10,9 +10,9 @@ By default, {VirtProductName} is installed with a single, internal pod network.
 
 You can create a Linux bridge network and attach a virtual machine (VM) to the network by performing the following steps:
 
-. xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nncp_virt-connecting-vm-to-linux-bridge[Create a Linux bridge node network configuration policy (NNCP)].
```
Comment (rh-tokeefe) [inline virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc:13]: This formatting is not supported. The xref links must follow the pattern outlined in the style guide or they will break on the customer portal.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

**Comment by rh-tokeefe** [comment]
@sbeskin-redhat thank you for your submission. It looks like a QE approval is still required on this PR. It is encouraged, but not required, to get SME and QE approval before requesting a peer review. However, QE approval must be obtained before merging a PR.

I am still onboarding for the OCP peer review process, so my peer with have to also look things over briefly. @bburt-rh please review this PR when you have a moment. Thank you.

**Comment by bburt-rh** [comment]
Thanks @rh-tokeefe! @sbeskin-redhat - Can you please update the xrefs in the PR per Tim's suggestion, and then ping us again here so that we can take a look? As long as the xrefs align with the repo guidelines, then we will be able to give it the `peer-review-done` label. Thank you!

**Comment by sbeskin-redhat** [comment]
@rh-tokeefe @bburt-rh 
Changing the xref format was the whole point of this Jira issue because the existing xrefs don't work on docs.openshift.com. They do, however, work on access.redhat.com. Maybe this PR is not needed at all?

@rh-tokeefe Why do you think this PR needs a QE review? The content is not changed here, it's just fixing xrefs.

@rh-tokeefe I am not sure I understand your comment about assemblies and modules. What content needs restructuring?

**Comment by rh-tokeefe** [comment]
@sbeskin-redhat I'm just sharing my understanding of the norms and practices associated with the OCP doc repo. Perhaps I'm misunderstanding some things. I'll invite @kalexand-rh and @bergerhoffer into the discussion, as they have deeper knowledge.

**Comment by bergerhoffer** [comment]
> @sbeskin-redhat I'm just sharing my understanding of the norms and practices associated with the OCP doc repo. Perhaps I'm misunderstanding some things. I'll invite @kalexand-rh and @bergerhoffer into the discussion, as they have deeper knowledge.

@rh-tokeefe is correct. All of our xrefs, no matter where they are to, must back up to the top level of the repository and go back down the path. So if they are broken on docs.openshift.com, there must be an issue with the current path.

I would also agree though that this doesn't require QE review if it's just fixing broken links.

**Comment by bergerhoffer** [comment]
It looks like @apinnick actually fixed this already in main and 4.15, but it wasn't cherry picked back to 4.14.

https://github.com/openshift/openshift-docs/pull/68978

I think we can just cherry pick her fix back to 4.14, and that should resolve the broken link on 4.14 too.

**Comment by rh-tokeefe** [comment]
Thank you @bergerhoffer. My prior understanding was that PRs always required QA review. I appreciate you clarifying this for me. 

@bburt-rh based on Andrea's comment I think we can move this to the next step in the process.

**Comment by bergerhoffer** [comment]
> Thank you @bergerhoffer. My prior understanding was that PRs always required QA review. I appreciate you clarifying this for me.
> 
> @bburt-rh based on Andrea's comment I think we can move this to the next step in the process.

@rh-tokeefe To clarify, any updates that are technical at all in nature must be QE reviewed. The exceptions to that are things that don't touch technical meaning, such as basic typos, broken links, formatting issues. I see that the peer review checklist doesn't mention these exceptions. I will look into making an update to clarify this!

As for moving this PR, I think we'll likely want to just close this PR and cherry pick Avital's PR to 4.14. But we can wait for @sbeskin-redhat to confirm that he's good with that plan.

**Comment by bburt-rh** [comment]
Thanks @bergerhoffer and @rh-tokeefe for your input on this PR.

@sbeskin-redhat - Andrea suggested closing this PR and cherry-picking Avital's changes to the older enterprise branches. WDYT? Feel free to close it if you agree.

File: virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc [hunk 2]
```diff
@@ -10,9 +10,9 @@ By default, {VirtProductName} is installed with a single, internal pod network.
...
-. Create a Linux bridge network attachment definition (NAD) by using the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-cli_virt-connecting-vm-to-linux-bridge[command line].
-. Configure the VM to recognize the NAD by using the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-vm-creating-nic-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge[command line].
+. xref:virt-creating-linux-bridge-nncp_virt-connecting-vm-to-linux-bridge[Create a Linux bridge node network configuration policy (NNCP)].
```
Comment (ctomasko) [inline virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc:13]: Please correct the xref formatting.   Should be:
xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-cli_virt-connecting-vm-to-linux-bridge[command line].

**Pull request closed without merging**